test/fixtures/dubhash/config.bat

```diff
@@ -0,0 +1,547 @@
+@echo off
+Rem Configure Bison for DJGPP.

```

Comment:
@robguinness-snyk (remember of replace this with an alternative fixture, as you raised with me)